.github/workflows/release.yml

```diff
@@ -54,44 +44,6 @@ jobs:
       - name: Copy Repository
         shell: bash
         run: |
-          # Create a temporary folder to clone the other repo
```

Comment:
I don't think moving this to justfile is a solution is better to have a separate script. and in general we can look for tools that that can run it locally for testing instead of moving everything.

Reply:
ok then I move it to a script.